data/World/Overworld.json

```diff
@@ -694,6 +694,25 @@
                 Gerudo_Membership_Card and can_ride_epona and Bow and at_day",
             "GF HBA 1500 Points": "
                 Gerudo_Membership_Card and can_ride_epona and Bow and at_day",
+            "GF Crate 1": "can_break_crate",
+            "GF Crate 2": "can_break_crate",
+            "GF Crate 3": "can_break_crate",
+            "GF Crate 4": "can_break_crate",
+            "GF Crate 5": "can_break_crate",
+            "GF Crate 6": "can_break_crate",
+            "GF HBA Crate 1":  "((is_adult and Gerudo_Membership_Card) or is_child) and can_break_crate",
```

Comment:
Maybe the archery range should be a separate region?

Reply:
There are maybe a couple other cases along these lines but I figured, it wasn't broke. If these kinds of things are still there next time I go through it all, I'll clean up some things with extra regions at that time, whenever that happens to be.